specs/simpleserialize.md

```diff
@@ -0,0 +1,230 @@
+# SimpleSerialize (SSZ) Spec
+
+***Work In Progress***
+
+This is the work in progress document to describe `simpleserialize`, the
+current selected serialization method for Ethereum 2.0 using the Beacon Chain.
+
+This document specifies the general information for serializing and
+deserializing objects and data types.
+
+## ToC
+
+* [About](#about)
+* [Terminology](#terminology)
+* [Constants](#constants)
+* [Overview](#overview)
+ + [Serialize/Encode](#serializeencode)
+   - [uint: 8/16/24/32/64/256](#uint-816243264256)
+   - [Address](#address)
+   - [Hash32](#hash32)
```

Comment:
With BLS we will have `Hash96` for the public keys maybe `Hash97` if it comes with a recovery bit.

Reply:
@mratsim interesting point! I'll leave this open for discussion if we should tentatively add it in or leave it until we have more clarity on the point?

I'll draft it up and maybe add it into a PR?

Reply:
a simple `Hash32 and other hash size` is probably enough at the moment